frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/personnel-popup/personnel-popup.component.html

```diff
@@ -10,7 +10,10 @@ <h5 class="popover-header">
     </h5>
     <div class="popover-body">
         <div class="mb-3">
-            <app-cater-capacity [type]="'personnel'" [canCaterFor]="personnel.canCaterFor"></app-cater-capacity>
+            <app-cater-capacity
+                [type]="'personnel'"
```

Comment:
Ah, so `[key]` is similar to `v-bind:key` or `:key` from vue in that the given value is interpreted as js instead of a string?

Reply:
> Ah, so `[key]` is similar to `v-bind:key` or `:key` from vue in that the given value is interpreted as js instead of a string?

Yes.

```
[key]="value"
```
could also be written as
``` 
key="{{ value }}"
```

But `value` can, in each case, only be a subset of JS (no template strings, arrow functions, ...).
There is probably also a difference between Vue and Angular in when and how often this expression is evaluated.